lib/runtime/wasmtime/instance.go

```diff
@@ -100,6 +100,11 @@ func (in *Instance) NetworkService() gssmrruntime.BasicNetwork {
 	return ctx.Network
 }
 
+// Exec calls the given function with the given data
+func (in *Instance) Exec(function string, data []byte) ([]byte, error) {
+	return in.exec(function, data)
+}
+
 func (in *Instance) exec(function string, data []byte) ([]byte, error) {
```

Comment:
Curious why you created a new func instead of just making the existing one public?

Reply:
no particular reason, I added the public func since I realized the w3f tester needed it, I could update it to be one func